### ISSUE TITLE
Manager Bug

### ISSUE THREAD

**Issue by Yushe** [spootTheLousy/saguaro#177]
I made a test account with the permissions: janitor_board,janitor,moderator,manager and I made sure that it was denied admin.

But for some reason it makes you basically a janitor:
On the admin post form you can't use capcodes (it says you are posting as: )
Says permission denied if you try to create a ban (needs to be shown on janitor_board) 
The only buttons you can see at the top are [return] [update] and [logout]

Basically it makes you a janitor it hides ips ect. This bug appears with manager and janitor not janitor_board but janitor_board does need to be changed.

**Comment by Apogate** [comment]
It's a matter of time before I just implement the user class

Chugging through learning PHP sessions right now, so deal with it for the time being.

**Comment by RePod** [comment]
``` PHP
session_start();
$_SESSION['apogate'] = 'sux';
```

**Comment by Apogate** [comment]
t-thanks dad
